apps/publisher/themes/default/js/taxonomy-browser.js

```diff
@@ -419,3 +434,24 @@ $(function () {
         }
     }
 });
+
+$(window).resize(function () {
```

Comment:
@splinter Added a comment and explained in #841 (c7e7f2fba36a308ff02c339614352b54567c54fa)